src/main/java/gov/cabinetoffice/gap/applybackend/web/SubmissionController.java

```diff
@@ -354,22 +357,29 @@ public ResponseEntity<GetNavigationParamsDto> getNextNavigationForQuestion(@Path
 
     @GetMapping("/{submissionId}/download-summary")
     public ResponseEntity<ByteArrayResource> exportSingleSubmission(
-            @PathVariable final UUID submissionId, HttpServletRequest request) throws Exception {
+            @PathVariable final UUID submissionId, HttpServletRequest request) {
         final String userSub = getUserIdFromSecurityContext();
         final String userEmail = grantApplicantService.getEmailById(userSub, request);
+        final Submission submission = submissionService.getSubmissionById(submissionId);
         
-        try (OdfTextDocument odt = submissionService.getSubmissionExport(submissionId, userEmail, userSub);
+        try (OdfTextDocument odt = submissionService.getSubmissionExport(submission, userEmail, userSub);
```

Comment:
Generally want to avoid putting logic like this in a controller. It's better placed in a service or Helper class